models/core/logs.sql

```diff
@@ -0,0 +1,63 @@
+{{
+    config(
+        materialized='incremental',
+        unique_key = 'log_id',
+        tags=['core', 'logs'],
+        cluster_by=['block_timestamp']
+    )
+}}
+
+with base_txs as (
+
+  select * from {{ ref("txs_deduped") }}
+
+),
+
+logs_raw as (
+
+    select
+
+        block_id,
+        block_timestamp,
+        tx_id as tx_hash,
+        tx:receipt:logs as full_logs
+
+    from base_txs
+    where {{ incremental_load_filter("block_timestamp") }}
```

Comment:
Can we move this where clause to the base_txs? I know snowflake is pretty smart and probably can optimize it, but I think its okay to have our incremental "where" in as early as possible, together with the base clause.. (dbt style guide also doesn't have such an example) @chuxinh what do you think?

Reply:
Yep, I'm not against that. Still keeps the CTE dependency structure in tact.